src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLogger.cs

```diff
@@ -40,7 +43,7 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
             }
             t_stringWriter ??= new StringWriter();
             LogEntry<TState> logEntry = new LogEntry<TState>(logLevel, _name, eventId, state, exception, formatter);
-            Formatter.Write(in logEntry, ScopeProvider, t_stringWriter);
+            Formatter!.Write(in logEntry, ScopeProvider, t_stringWriter);
```

Comment:
Since this type is internal, can we just change the ctor to take the `ConsoleFormatter` and make the `Formatter` property non-nullable?